wurst/systems/crafting/QuickMakeToolTips.wurst

```diff
@@ -177,7 +177,7 @@ public let SCROLL_FIREBALL_RECIPE       = "a Fireball Spell Scroll. Can cast fir
 public let SCROLL_CYCLONE_RECIPE        = "a Cyclone Spell Scroll. Can cast cyclone on a target enemy."+makeToolTipDuration(5, 10, 30)+"\n\n1x "+WIND+" + 1x "+MC
 public let SCROLL_LIVING_DEAD_RECIPE    = "a Living Dead Spell Scroll. Can summon 2 skeletons fighting for you."+makeToolTipDuration(30, 30, 60)+"\n\n1x "+BONE+" + 1x "+MC
 public let SCROLL_TSUNAMI_RECIPE        = "a Tsunami Spell Scroll. Can cast a tsunami, dealing 25 damage. Good for destroying base defenses."+makeToolTipCooldown(25)+"\n\n1x "+WATER+" + 1x "+MC
-public let SCROLL_SWIFTNESS_RECIPE      = "a Swiftness Spell Scroll. Can increase movement speed of surrounding allies to maximum."+makeToolTipDuration(7, 7, 35)+"\n\n1x "+ANABOLIC+" + 1x "+MC
+public let SCROLL_HASTE_RECIPE      = "a HASTE Spell Scroll. Can increase movement speed of surrounding allies to maximum."+makeToolTipDuration(7, 7, 35)+"\n\n1x "+ANABOLIC+" + 1x "+MC
```

Comment:
This didn't play nicely with your find + replace. The operator isn't lined up and `HASTE` is capitalized.